membership.lua

```diff
@@ -362,9 +370,10 @@ local function handle_message_step()
 end
 
 local function handle_message_loop()
-    local sock = _sock
-    while sock == _sock do
+    fiber.name('swim.handle')
```

Comment:
I suggest to use `membership` namespace. Because module is called `membership`... It could confuse users because currently Tarantool has built-in "swim" module.

Reply:
Ok